crates/openapi/src/routes/payments.rs

```diff
@@ -599,6 +599,21 @@ pub fn payments_dynamic_tax_calculation() {}
 
 pub fn payments_post_session_tokens() {}
 
+/// Payments - Update Metadata
+#[utoipa::path(
+    post,
+    path = "/payments/{payment_id}/update_metadata",
+    request_body=PaymentsUpdateMetadataRequest,
+    responses(
+        (status = 200, description = "Update Metadata is done", body = PaymentsUpdateMetadataResponse),
+        (status = 400, description = "Missing mandatory fields")
+    ),
+    tag = "Payments",
+    operation_id = "Create Update Metadata for a Payment",
+    security(("publishable_key" = []))
```

Comment:
Is this supposed to be API key authentication or publishable key? I noticed that the route handler in the `router` crate uses API key authentication.